### PR TITLE
updated alumni sorted, hidden neural net, stipend update

### PR DESCRIPTION
**Pull request by chiranthans23**
- Added alumni officers and sorted them in reverse order based on the date, chair picture fix
- Neural nets page is hidden with an extra field
- Stipends of TA, RA, CM, and CGs are updated
- The number of hours section is hidden
 
image
image
image

### CODE REVIEW

**Comment by chiranthans23** [comment]
image

updated stipend and benifits

File: _data/menu.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 - name: The Neural Network
   url: /neural-network
+  visible: hide
```
Comment (Dreycey) [inline _data/menu.yml:20]: Perfect

File: _data/officers.yml [hunk 2]
```diff
@@ -3,8 +3,8 @@
 chair:
     name: James Luther
     contact: James.Luther
-    photo: assets/img/officers/james_luther.jpg
-    pronouns: She/Her/Hers
+    photo: assets/img/officers/james_luther.jpeg
+    pronouns: He/Him/His
```
Comment (Dreycey) [inline _data/officers.yml:7]: NICE catch

**Review by Dreycey — APPROVED** [review]

**Comment by Dreycey** [comment]
Everything looks great. These updates will offer a huge visual improvement!

**Pull request merged**